app/middleware/request_validations.py

```diff
@@ -0,0 +1,70 @@
+"""This module contains Middleware class"""
+
+
+class Middleware:
+    """
+    This class handles code submission validation for the application.
+    The Middleware class validates incoming code submissions and ensures
+    they meet the necessary criteria.
+    """
+
+    def __init__(self, data_dict):
+        """
+        Initializes an instance of Middleware for request validation.
+
+        Parameters:
+        - data_dict (dict):
+        A dictionary containing request data to be validated.
+        """
+        self.code = data_dict["code"]
+        self.input_data = data_dict["input"]
+        self.output_data = data_dict["output"]
+        self.time_limit = data_dict["time_limit"]
+        self.memory_limit = data_dict["memory_limit"]
+        self.language = data_dict["language"]
+
+    def validate_code(self):
+        """
+        Validating if the language name is standardized
+        if the code is not empty
+        if the time limit is reasonable
+        """
+        if not self.standardize_language_name():
+            return False
+
+        if self.code == "":
+            return False
+
+        if self.time_limit > 10:
```

Comment:
```suggestion
        if self.time_limit > 10000:
```

La unidad que vamos a usar es de ms

Reply:
resolve in https://github.com/MinervaLMS/judge-api/commit/818f80c227114367814f43b7fdb5435c8b8618c6